turbopack/crates/turbo-tasks-backend/Cargo.toml

```diff
@@ -23,14 +23,14 @@ arc-swap = { version = "1.7.1" }
 async-trait = { workspace = true }
 auto-hash-map = { workspace = true }
 byteorder = "1.5.0"
-dashmap = { workspace = true, features = ["raw-api"]}
+dashmap = { workspace = true, features = ["raw-api"] }
 either = { workspace = true }
 hashbrown = { workspace = true, features = ["raw"] }
 indexmap = { workspace = true }
 lmdb-rkv = "0.14.0"
 once_cell = { workspace = true }
 parking_lot = { workspace = true }
-pot = "3.0.0"
+bincode = "2.0.0-rc.3"
```

Comment:
This pot/bincode change probably shouldn't be in this PR

Reply:
oops you're right. must have popped 1 too many stashes...